vsts/pipelines/baseImages/dotnetcore.yml

```diff
@@ -14,19 +14,20 @@ jobs:
     artifactsFileName: dotnetcore-runtimeimage-bases-buster.txt
     jobName: Build_DotNetCore_BaseImage_Buster
 
-- template: ../templates/_buildimageBasesJobTemplate.yml
-  parameters:
-    displayName: Build DotNetCore runtime bullseye base images
-    scriptPath: ./build/buildRunTimeImageBases.sh
-    imageDir: dotnetcore
-    imageDebianFlavor: bullseye
-    artifactsFileName: dotnetcore-runtimeimage-bases-bullseye.txt
-    jobName: Build_DotNetCore_BaseImage_Bullseye
+# Temporarily commenting this out to skip building bases for bullseye images.
```

Comment:
If you didn't already, please consider creating a work item so that we revert this once the image is generated. If you're tracking that in your TODO list and will revert it yourself, please feel free to ignore.